llvm/test/Transforms/Attributor/returned-dl-sentinel.ll

```diff
@@ -0,0 +1,1421 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --function-signature --check-attributes --check-globals
+; RUN: opt -aa-pipeline=basic-aa -passes=attributor -attributor-manifest-internal  -attributor-annotate-decl-cs  -S < %s | FileCheck %s --check-prefixes=CHECK,TUNIT
```

Comment:
Same here, all these attributor tests seem to just be copies of existing tests with the datalayout applied. This needs to be more targeted (and ideally would have 2 identical copies of functions in the same test file, with different property address spaces)

Reply:
> Same here, all these attributor tests seem to just be copies of existing tests with the datalayout applied. This needs to be more targeted (and ideally would have 2 identical copies of functions in the same test file, with different property address spaces)

Updated tests with and without datalayout string changes. added datalayout string to opt run cmd.